.github/workflows/update-demo-rooks.yaml

```diff
@@ -0,0 +1,60 @@
+name: Update Go Demo Rooks
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v[0-9].[0-9]+.*'
+
+jobs:
+  update_demo:
+    runs-on: ubuntu-latest
+    env:
+      GO_ROOK_VERSION:  ${{ github.ref_name }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          path: main
+      - name: Checkout microservices-demo repo
+        uses: actions/checkout@v3
+        with:
+          repository: Rookout/microservices-demo
+          token: ${{ secrets.SONARIO_GITHUB_TOKEN }}
+          path: microservices-demo
+      - name: Checkout tutorial-go repo
+        uses: actions/checkout@v3
+        with:
+          repository: Rookout/tutorial-go
+          token: ${{ secrets.SONARIO_GITHUB_TOKEN }}
+          path: tutorial-go
+      - name: Update tutorial-go
+        working-directory: ./tutorial-go
+        run: |
+          go get -u github.com/Rookout/GoSDK@${{env.GO_ROOK_VERSION}}
+          git config --global user.email "sonario@rookout.com"
+          git config --global user.name "sonariorobot"
+          git remote -v
+          git add -A
+          git status
+          git commit -m "Update GoSDK to: {{env.GO_ROOK_VERSION}}"
+          git push
+      - name: Update microservices-demo
+        working-directory: ./microservices-demo
+        run: |
+          cd src/checkoutservice
+          go get -u github.com/Rookout/GoSDK@${{env.GO_ROOK_VERSION}}
+          go mod tidy
+          cd ../../src/frontend && go get -u github.com/Rookout/GoSDK@${{env.GO_ROOK_VERSION}}
+          go mod tidy
+          cd ../../src/productcatalogservice && go get -u github.com/Rookout/GoSDK@${{env.GO_ROOK_VERSION}}
+          go mod tidy
+          cd ../../src/shippingservice && go get -u github.com/Rookout/GoSDK@${{env.GO_ROOK_VERSION}}
+          go mod tidy
+          git config --global user.email "sonario@rookout.com"
```

Comment:
@alexeygutkin Shouldn't you do a cd .. && cd .. ?

Reply:
@Urook not necessary

Reply:
ok